### DISCUSSION TITLE
What ALPINE_DATABASE_DRIVER setting should i be using to connect to a Aurora PostgreSQL in AWS ?

### DISCUSSION THREAD

**Discussion by whiteninja76** [DependencyTrack/dependency-track#3046] [Q&A] [answered]
I'm trying to spin up ECS task using the 'dependencytrack/apiserver' container but not sure what the setting should be to access the rds database correctly.

The env's on the container are set to 
ALPINE_DATABASE_DRIVER: org.postgresql.Driver
ALPINE_DATABASE_MODE: external
ALPINE_DATABASE_URL: (dbname).(clusterid).eu-west-2.rds.amazonaws.com

CloudWatch Logs :
 java.sql.SQLException: No suitable driver found for (dbname).(clusterid).eu-west-2.rds.amazonaws.com

Am i using the wrong driver, do i need to load in a different driver ?

**Comment by syalioune** [comment, marked as answer]
Hello @whiteninja76 
What is the value of your `ALPINE_DATABASE_URL` ? It should be prefixed by `jdbc:postgresql://` and the `-cluster.cluster-` value seems odd to me. Almost as if there's a variable substitution that resolve to empty string.

**Reply by whiteninja76** [reply]
HI syalioune I have updated the original post , hadn't noticed my numerous typos in my question . 
Are you saying the ALPINE_DATABASE_URL should be jdbc:postgresql://(rdsDatbaseEndpoint) ?

**Reply by syalioune** [reply]
yes prefixing it should do the job
